www_src/pages/project/project.jsx

```diff
@@ -331,21 +392,18 @@ var Project = React.createClass({
               page,
               selected: page.id === this.state.selectedEl,
               transform: this.cartesian.getTransform(page.coords),
-              onClick: this.selectPage(page)
+              onClick: isPlayOnly ? this.zoomToPage.bind(this, page.coords) : this.selectPage.bind(this, page)
```

Comment:
I feel kind of gross doing this, but otherwise the methods get immediately invoked on every render... :sob: